xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1123,11 +1123,45 @@ bool CGUIWindowVideoBase::OnPlayAndQueueMedia(const CFileItemPtr &item, std::str
   return CGUIMediaWindow::OnPlayAndQueueMedia(movieItem, player);
 }
 
-void CGUIWindowVideoBase::PlayMovie(const CFileItem *item, const std::string &player)
+void CGUIWindowVideoBase::PlayMovie(CFileItem *item, const std::string &player)
```

Comment:
you are not changing this pointer so why remove the const?

Reply:
I need call SetPath to change the movie version file, which will violate this const.